openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -45,7 +45,7 @@
 
         this._columns = [
             {
-                name: 'parent_name', label: gettext("Parent Section"), label_summary: gettext("Parent Sections"),
+                name: 'parent_name', label: gettext("Unit Name"), label_summary: gettext("Units"),
```

Comment:
@srpearce @grantgoodmanedX Here's the doc change we were discussing in https://github.com/edx/edx-platform/pull/14562, so it'll be live when the main feature lands 👍